### PR TITLE
doc: procedure for new release branch from main

### LINKED ISSUE #702: Document new minor release process
Integrate the release procedure with instructions on how to create a new minor release.

### PR DESCRIPTION
**Pull request by gbartolini**
Closes #702

Signed-off-by: Gabriele Bartolini <gabriele.bartolini@enterprisedb.com>

### CODE REVIEW

**Review by armru — APPROVED** [review]

File: contribute/release_procedure.md [hunk 1]
```diff
@@ -42,6 +42,18 @@ Sometimes, bug fixes might originate in the release branch as well.
 Release notes for patch/security versions are maintained in the release branch
 directly.
 
+### Creating a new release branch
```
Comment (armru) [inline contribute/release_procedure.md:45]: I would replace with `Creating a new  minor release branch from main`

File: contribute/release_procedure.md [hunk 2]
```diff
@@ -68,6 +80,11 @@ activities:
 Once the code in the release branch is stable and ready to be released, you can
 proceed with the supervised process.
 
+**IMPORTANT:** You need to operate on an existing release branch. If you are
+releasing a new minor version, you must create the new release branch
+immediately before proceeding with the instructions below. In order to create
+a new release branch, see "Creating a new release branch" above.
```
Comment (armru) [inline contribute/release_procedure.md:86]: this line would have to be fixed if we change the paragraph title

File: contribute/release_procedure.md (hunk 1 above)
Comment (gbartolini) [inline contribute/release_procedure.md:45]:
```suggestion
### Creating a new release branch from main
```

File: contribute/release_procedure.md (hunk 2 above)
Comment (gbartolini) [inline contribute/release_procedure.md:86]:
```suggestion
a new release branch, see "Creating a new release branch from main" above.
```

**Review by mnencia — APPROVED** [review]

**Pull request merged**